### ISSUE TITLE
Freerouting plugin stopped working

### ISSUE THREAD

**Issue by dp1044** [freerouting/freerouting#360]
**Describe the bug**
Yesterday morning I finished a project and freerouting worked fine.  Yesterday Afternoon it stopped working.
Freerouting plugin for kicad 7 and 8 does not complete routing all the traces.

**To Reproduce**
1. Go to kicad 7/8
2. Click on Freerouting plugin
3. click on autoroute
4. It routes only a few then quits back to kicad with only those routes connected

**Expected behavior**
To have freerouting finish all traces before if finishes

**Screenshots**
Here are shots of the board before and after autorouting.
![Screenshot 2024-10-16 185950](https://github.com/user-attachments/assets/1571a254-ed44-4d22-b2a9-39d405ed42a9)
![Screenshot 2024-10-16 190150](https://github.com/user-attachments/assets/3ba86bbd-1c0a-4d44-b413-cb1ebc7a1309)

**Environment**
 - OS: Windows 11, 64-bit
 - Freerouting version: 1.8 and 1.9
 - EDA: Kicad 7 and 8.0.6

**Command line arguments**
N/A

**Project files**
zip of the project
[BattMonitor.zip](https://github.com/user-attachments/files/17403570/BattMonitor.zip)

**Logs**
no errors or logs

**Additional context**
I have tried the other suggestions, uninstall and reinstall of the plugin, kicad and jre. Nothing was changed on my setup or installed between it working and stopping.

**Comment by boromyr** [comment]
same issue with version 2

**Comment by boromyr** [comment]
Does anyone have this problem? For me FR no longer works completely, it routes 5% of the routes and then closes.

**Comment by anomaly256** [comment]
Yes I am having exactly the same issue.  A few traces created, everything else 'failed' but no reason given.

![image](https://github.com/user-attachments/assets/219f0e4c-954a-4eb3-80de-7a9fd3c5476c)

**Comment by anomaly256** [comment]
@boromyr actually, for my case, I've found the culprit - the footprints for the sparkfun/adafruit PCBs I'm using have the rectangular outline on the edgecuts layer, causing freerouting and kicad itself to fail to connect routes to those pads.  Moving the outline to the silkscreen layer fixed the auto routing.

I'm not sure if U1 in your case is a custom footprint or not but it might be worth checking which layer things are on and making sure they're correct, or making a new custom footprint.

**Comment by boromyr** [comment]
![image](https://github.com/user-attachments/assets/e745b5a4-4e12-41e3-9bad-89595c48787c)
In my case everything is partially routed, it's not a problem with the single footprint. Where could I find more specific log files?
